### PR TITLE
Paraguay holidays update

### PR DESCRIPTION
**Pull request by KJhellico**
- added special holidays
- fixed observance calculation

### CODE REVIEW

File: holidays/countries/paraguay.py [hunk 1]
```diff
@@ -27,82 +39,201 @@ class Paraguay(HolidayBase):
...
-            pass
-        elif date(year, MAR, 1).weekday() >= WED:
-            self[date(year, MAR, 1) + rd(weekday=MO(+1))] = name
+        dates_obs = {
+            2013: (MAR, 4),
+            2016: (FEB, 29),
+            2018: (FEB, 26),
+            2022: (FEB, 28),
+        }
+        if year in dates_obs:
```
Comment (arkid15r) [inline holidays/countries/paraguay.py:169]: Optional (here and below): 
```
self[date(year, *dates_obs.get(year, (MAR, 1)))] = "Día de los Héroes de la Patria [Patriots Day]"
```

**Review by arkid15r — COMMENTED** [review]
LGTM

File: holidays/countries/paraguay.py (hunk 1 above)
Comment (KJhellico) [reply holidays/countries/paraguay.py]: Thank you! That's why I like Python, for things like this :)

**Comment by dr-prodigy** [comment]
👍 👍

**Pull request merged**